Sources/SwifterSwift/SwiftStdlib/KeyedDecodingContainer.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  KeyedDecodingContainer.swift
+//  SwifterSwift
+//
+//  Created by Francesco Deliro on 23/10/2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+#if canImport(Foundation)
```

Comment:
This needs to be placed around the functions themselves, since they're what will actually give you compile errors when `Foundation` isn't available.